packages/react/src/internal/field-label/FieldLabel.module.css

```diff
@@ -5,3 +5,9 @@
 .hidden {
   composes: hiddenFromScreen from 'hds-core/lib/utils/helpers.css';
```

Comment:
Would it be possible to change the file from `css` to `scss` and use the `%hiddenFromScreen` helper class from `_common.scss` instead? We've tried to get rid of `composes` where possible, as it seems like it doesn't always play nicely with rollup and causes some duplicate code in the bundle.
Same thing with `.label`. Could the label styles be copied here from `text-input.css`?

Reply:
Makes sense. This has now been changed.